backend/Cargo.toml

```diff
@@ -58,6 +58,7 @@ oauth2 = { version = "5.0.0", features = ["reqwest"] }
 lettre = "0.11.13"
 dashmap = "6.1.0"
 validator = { version = "0.20.0", features = ["derive"] }
+text-template = "0.1.0"
```

Comment:
Do we really want to want use unmaintained library not updated for 6 years, with no readme hosted on gitlab. I was thinking about something more powerful with templates, so we could easily create like components etc

Reply:
I was thinking about libraries like tera or even handlebars `handlebars` with html escaping, basic logic like if statements, templates etc.